ogcapi-stable/ogcapi-common/src/main/java/de/ii/ldproxy/ogcapi/infra/rest/OgcApiExceptionFormatJson.java

```diff
@@ -25,18 +23,17 @@
     @Requires
     SchemaGenerator schemaGenerator;
```

Comment:
JSON responses fail because the SchemaGenerator is always null. Any ideas why this could be happening?

Reply:
I think, because SchemaGenerator is not ready when the class is created. I have changed it to create the simple RFC 7807 schema manually.